runtime/src/lib.rs

```diff
@@ -188,6 +188,22 @@ impl session::Trait for Runtime {
 	type Event = Event;
 }
 
+/// Converter for currencies to votes.
+pub struct CurrencyToVoteHandler;
```

Comment:
cc @Kianenigma copied from substrate-node. is it ok?

Reply:
Yup. This is also the new version. Worth keeping an eye on Staking in the next test-net runtime upgrade.

Reply:
The PR that would affect the next runtime upgrade is https://github.com/paritytech/polkadot/pull/247 (against poc-4 branch)